contracts/controller/MarketEnabled.sol

```diff
@@ -76,10 +77,11 @@ contract MarketEnabled is NutzEnabled {
     return amountBabz;
   }
 
-  function sell(address _from, uint256 _amountBabz) public onlyNutz whenNotPaused {
+  function sell(address _from, uint256 _price, uint256 _amountBabz) public onlyNutz whenNotPaused {
     uint256 effectiveFloor = floor();
     uint256 INFINITY = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
     require(effectiveFloor != INFINITY);
+    require(_price == effectiveFloor);
```

Comment:
is there any way we can tell the caller what was the reason of the failure (e.g. price changed) ? Maybe an event?